helios-services/src/main/java/com/spotify/helios/agent/AddExtraHostContainerDecorator.java

```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.helios.agent;
 
+import com.google.common.base.Optional;
```

Comment:
Everywhere in Helios uses the Guava class. I think because we started at Java 7?